### PR TITLE
Add bytecode verification

### PR DESCRIPTION
**Pull request by madlabman**
- Update and fix existing workflows
- Add `verify-bytecode` workflow

@TheDZhon, please take a look.

### CODE REVIEW

File: .github/workflows/verify-bytecode.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Setup node.js version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Cache node_modules
```
Comment (skhomuti) [inline .github/workflows/verify-bytecode.yml:23]: is it really needed? you can use `cache` arg from actions/setup-node action as from another workflows

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: .github/workflows/verify-bytecode.yml (hunk 1 above)
Comment (madlabman) [reply .github/workflows/verify-bytecode.yml:23]: Takeaway from `setup-node` action README:

> Note: The action does not cache node_modules

**Comment by TheDZhon** [comment]
Hey, will take a look, but the repo owner is @team-intergrations :pray: 
cc: @Psirex, @bulbozaur, @rkolpakov

**Review by dgusakov — APPROVED** [review]

File: artifacts-arb.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  },
+  {
+    "artifactPath": "artifacts/contracts/token/ERC20Bridged.sol/ERC20Bridged.json",
+    "sourcePath": "contracts/token/ERC20Bridged.sol",
+    "name": "WstETH ERC20Bridged",
+    "address": "0x0fBcbaEA96Ce0cF7Ee00A8c19c3ab6f5Dc8E1921",
+    "txHash": "0x517ea5c206473d2116324939ab42f93ca1079a6388c0db41684e926e36e1a411"
+  },
+  {
+    "artifactPath": "utils/arbitrum/artifacts/L2GatewayRouter.json",
```
Comment (Psirex) [inline artifacts-arb.json:29]: We don't control this contract, so better exclude it from checks. Also, our repo doesn't contain the `L2GatewayRouter.sol` contract not sure how critical it's for the bytecode validator.

File: artifacts-eth.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  {
+    "artifactPath": "artifacts/contracts/optimism/L1ERC20TokenBridge.sol/L1ERC20TokenBridge.json",
+    "sourcePath": "contracts/optimism/L1ERC20TokenBridge.sol",
+    "name": "L1ERC20TokenBridge",
+    "address": "0x29C5c51A031165CE62F964966A6399b81165EFA4",
+    "txHash": "0xfd7c2ee015321e1dcd9f0f0f6b2b15a1731087ba483190a1c8d616e3ffbf0323"
+  },
+  {
+    "artifactPath": "utils/arbitrum/artifacts/L1GatewayRouter.json",
```
Comment (Psirex) [inline artifacts-eth.json:31]: Same as for the `L2GatewayRouter`

File: artifacts-arb.json (hunk 2 above)
Comment (madlabman) [reply artifacts-arb.json]: Done, thx!

File: artifacts-eth.json (hunk 3 above)
Comment (madlabman) [reply artifacts-eth.json]: Done.

**Pull request merged**